.github/workflows/rss-runner.yaml

```
@@ -23,20 +23,72 @@ jobs:
           chmod +x setup.sh
           ./setup.sh
       
-      - name: Update runningCSV.csv with Events 
+      - name: Update runningCSV.csv with Jobs 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)